index.html

```diff
@@ -1,413 +1,308 @@
+    that it’s all completely free of charge." />
+  <meta name="keywords"
+    content="New Devs on the Block, Vienna, Workshop, HTML, CSS, Martin Splitt, Eva Lettner, Cassie Evans, Nico König, Ramón Huidobro" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+  <link rel="manifest" href="site.webmanifest" />
+  <link rel="mask-icon" href="/images/safari-pinned-tab.svg" color="#5bbad5" />
+  <meta name="msapplication-TileColor" content="#da532c" />
+  <meta name="theme-color" content="#ffffff" />
+  <link href="https://fonts.googleapis.com/css?family=Lato|Oswald" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
+  <link rel="stylesheet" href="styles.css" />
+  <title>NEW DEVS ON THE BLOCK</title>
+</head>
+
+<body>
+  <header id="top">
+    <nav>
+      <a href="#schedule">Schedule</a>
+      <a href="#workshops">Workshops & Register</a>
+      <a href="#coaches">Coaches</a>
+      <a href="#partners">Partners</a>
+    </nav>
+    <img src="images/logo.png" class="logo" alt="kickass new devs on the block logo" />
+  </header>
+  <main>
+    <article id="about">
+      <h2>About us</h2>
+      <p>
+        Have you always wanted to build your first own website, but somehow, you didn't know how to get started? Maybe
+        because you thought you were not that techie after all? Then these two days are precisely for you! We will take
+        the
+        first few steps down the web development road together.
+      </p>
+      <p>
+        NEW DEVS ON THE BLOCK is a two-day workshop for front-end development made and supported by the beautiful people
+        of the web community.
+      </p>
+      <p>
+        We will help you learn the fundamental building blocks of websites and guide you through coding your first
+        website yourself.
+      </p>
+    </article>
+
+    <article id="schedule">
+      <h2>Schedule</h2>
+      <p>
+        The workshop is divided into different topics that build on top of each other.
+        Each topic contains a theoretical part, where we go over some slides, and a coding part, where you will turn
+        theory into code.
+      </p>
+
+      <h3>Topics</h3>
+      <p>You will learn the following things</p>
+      <ul>
+        <li>History of the Web</li>
+        <li>Create the basic structure of a website</li>
+        <li>Add styling (i.e. fonts and colors) to your website</li>
+        <li>How to add images to your website</li>
+        <li>How to create layouts</li>
+        <li>How to add minimal animations</li>
+        <li>Use JavaScript to create a image gallery</li>
```

Comment:
```suggestion
        <li>History of the Web</li>
        <li>Basic structure of a website</li>
        <li>Styling (i.e. fonts and colors) your website</li>
        <li>Working with images</li>
        <li>Creating layouts</li>
        <li>Minimal animations</li>
        <li>Rotating image gallery</li>
```

I was trying to make it shorter/read a bit better. Also didn't mention JS because we have all of the technical bits below!